stream-chat-android-client/src/main/java/io/getstream/chat/android/client/setup/InitializationCoordinator.kt

```diff
@@ -28,20 +34,65 @@ public class InitializationCoordinator private constructor() {
 
     private val userDisconnectedListeners: MutableList<(User?) -> Unit> = mutableListOf()
 
+    /**
+     * Listeners which will get invoked when the web socket goes through connection changes.
+     */
+    private val onSocketConnectionStateChangedListener: MutableList<(ConnectionState) -> Unit> = mutableListOf()
```

Comment:
```suggestion
    private val onSocketConnectionStateChangedListeners: MutableList<(ConnectionState) -> Unit> = mutableListOf()
```

Reply:
This will be removed as per our slack discussion here: https://getstream.slack.com/archives/C02AA5LP0SK/p1659107529380459